server/services/mi-light/lib/light/light.setValue.js

```diff
@@ -0,0 +1,60 @@
+const { DEVICE_FEATURE_TYPES } = require('../../../../utils/constants');
+const logger = require('../../../../utils/logger');
+const { parseExternalId } = require('../utils/parseExternalId');
+const { NotFoundError } = require('../../../../utils/coreErrors');
+
+/**
+ * @description Change value of a milight zone
+ * @param {Object} device - The device to control.
+ * @param {Object} deviceFeature - The binary deviceFeature to control.
+ * @param {string|number} value - The new value.
+ * @example
+ * setValue(device, deviceFeature, value);
+ */
+async function setValue(device, deviceFeature, value) {
+  const { zoneId, bridgeMac } = parseExternalId(device.external_id);
+  // transform mac : adding ":"
+  let macAddressStdFormat = '';
+  for (let i = 0; i < 11; i += 2) {
+    macAddressStdFormat += bridgeMac.slice(i, i + 2);
+    if (i !== 10) {
+      macAddressStdFormat += ':';
+    }
+  }
+  const bridge = this.bridgesByMac.get(macAddressStdFormat.trim());
+
+  if (!bridge) {
+    throw new NotFoundError(`BRIDGE_NOT_FOUND`);
```

Comment:
Please review error message.

Reply:
I will change the message error : `"No bridge saved in Gladys with the following mac address : ${macAddressStdFormat.trim()}"`